receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -263,3 +272,72 @@ func createNodeAndResource(job, instance, scheme string) (*commonpb.Node, *resou
 	}
 	return node, resource
 }
+
+func fixStaleMetrics(md *pdata.Metrics) {
```

Comment:
Are the current tests sufficient to cover the cases from this function?

Reply:
I believe so.  There are end-to-end test for all aggregation types that include responses that should induce staleness markers.  The tests that are enabled in this PR ensure that those staleness markers are generated and that start time adjustment is correctly handled when they are present.